yt_dlp/extractor/zingmp3.py

```diff
@@ -35,13 +63,14 @@ def _extract_item(self, item, fatal):
                                 r'^(\d+)p', res, 'resolution', default=None)),
                         })
             else:
-                formats.append({
-                    'ext': 'mp3',
-                    'format_id': k,
-                    'tbr': int_or_none(k),
-                    'url': self._proto_relative_url(v),
-                    'vcodec': 'none',
-                })
+                if v.startswith("http"):
+                    formats.append({
+                        'ext': 'mp3',
+                        'format_id': k,
+                        'tbr': int_or_none(k),
+                        'url': self._proto_relative_url(v),
+                        'vcodec': 'none',
+                    })
```

Comment:
`_proto_relative_url` appends `http:`, so it should be left unchanged

Reply:
@Lesmiscore Oh, sr that's my bad, i should use `v == "VIP"`, because the response sometime like this {'128': 'https://mp3-s1-zmp3.zadn.vn/.../*~hmac=1246344681864cc9ea6a517b57e8fe7d&fs=MTY0NTgwNzY5NjE0Nnx3ZWJWNnwwfDE0LjE2MC45LjEy', '320': 'VIP'}

Reply:
@Lesmiscore Done. Ths you

Reply:
Make sure to resolve conversation when you're done